examples/docker/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3"
+services:
+  sub:
+    build:
+      context: ./sub  # Docker context from folder of this file; needed to include requirement.txt
+      dockerfile: Dockerfile
+    ports:
+      - "5550:5550"  # map container interal 5550 port to publicly accessible 5550 port
```

Comment:
why 5550 and not 5551 as the defaults would suggest?

Reply:
It has been a while ago I wrote this, but I think I did this so you could both test the Docker and non-Docker version at the same time. If the port value is the same, and you start them both, one of the 2 cannot bind.